### ISSUE TITLE
Feature Request: support version specifiers

### ISSUE THREAD

**Issue by junosuarez** [zkat/npx#9]
I want to be able to add a version specifier to request a particular version of a command. This could be useful either for checking out beta tagged versions that aren't yet "latest", or for going back in time and checking that something works with a specific prior version. It also makes it feel more npm-like.

right now, I get:
```sh
> npx standard@5 --version
6.0.4
```

and instead it would be awesome to get:
```sh
> npx standard@5 --version
5.4.1
```

(also, since `npm info standard --json | js .version` is 10.0.2, I'm not sure why I'm getting 6.0.4 other than assuming I've got it cached somewhere on my machine - but that's a different issue)

**Comment by zkat** [comment]
This should already work. Maybe update `npx`?

Another reason might be if you have `standard` installed globally or in the current project, in which case it *just runs* that binary. I guess I should force an install if you provide a `--package` or a `@version` specifier.

Here it is, without a global `standard`, though:

<img width="471" alt="screen shot 2017-05-30 at 15 40 49" src="https://cloud.githubusercontent.com/assets/17535/26608050/67d3bdda-454e-11e7-8ab8-7f507411f51d.png">

**Comment by junosuarez** [comment]
Ah, that was totally it- I had standard@6.0.4 installed globally.

> I guess I should force an install if you provide a --package or a @version specifier.

This would more or less match the behavior I expected. The "run globally installed version if available" is somewhat surprising.

**Comment by zkat** [comment]
This is available as of `npx@2.0.0`

**Comment by junosuarez** [comment]
💟
